cms/static/js/spec/views/group_configuration_spec.js

```diff
@@ -90,7 +87,7 @@ define([
         delete window.course;
     });
 
-    describe('GroupConfigurationDetails', function() {
+    describe('GroupConfigurationDetailsView', function() {
```

Comment:
I would find it helpful if this text made it clear that this is the details view for experiment group configurations.